src/gui/app.cpp

```diff
@@ -201,6 +207,10 @@ void App::processCliOptions(QCommandLineParser &parser, const QStringList& argum
 		parser.showHelp();
 	}
 
+	if (parser.isSet("version")) {
+		parser.showVersion();
```

Comment:
Is this check and the call to showVersion necessary? Reading the docs it seemed the call to QCommandLineParser::process would handle this case.